io.openems.edge.meter.bcontrol.em300/src/io/openems/edge/meter/bcontrol/em300/MeterBControlEM300Impl.java

```diff
@@ -130,43 +133,75 @@ protected ModbusProtocol defineModbusProtocol() throws OpenemsException {
 						m(MeterBControlEM300.ChannelId.REACTIVE_POWER_L2_POS, new UnsignedDoublewordElement(84),
 								SCALE_FACTOR_MINUS_1),
 						m(MeterBControlEM300.ChannelId.REACTIVE_POWER_L2_NEG, new UnsignedDoublewordElement(86),
-								SCALE_FACTOR_MINUS_1)),
+								SCALE_FACTOR_MINUS_1),
+
+						new DummyRegisterElement(88, 99),
 
-				new FC3ReadRegistersTask(100, Priority.HIGH,
 						m(ElectricityMeter.ChannelId.CURRENT_L2, new UnsignedDoublewordElement(100)),
-						m(ElectricityMeter.ChannelId.VOLTAGE_L2, new UnsignedDoublewordElement(102))),
+						m(ElectricityMeter.ChannelId.VOLTAGE_L2, new UnsignedDoublewordElement(102)),
+
+						new DummyRegisterElement(104, 119),
 
-				new FC3ReadRegistersTask(120, Priority.HIGH,
 						m(MeterBControlEM300.ChannelId.ACTIVE_POWER_L3_POS, new UnsignedDoublewordElement(120),
 								SCALE_FACTOR_MINUS_1),
 						m(MeterBControlEM300.ChannelId.ACTIVE_POWER_L3_NEG, new UnsignedDoublewordElement(122),
 								SCALE_FACTOR_MINUS_1),
 						m(MeterBControlEM300.ChannelId.REACTIVE_POWER_L3_POS, new UnsignedDoublewordElement(124),
 								SCALE_FACTOR_MINUS_1),
 						m(MeterBControlEM300.ChannelId.REACTIVE_POWER_L3_NEG, new UnsignedDoublewordElement(126),
-								SCALE_FACTOR_MINUS_1)),
+								SCALE_FACTOR_MINUS_1),
+
+						new DummyRegisterElement(128, 139),
 
-				new FC3ReadRegistersTask(140, Priority.HIGH,
 						m(ElectricityMeter.ChannelId.CURRENT_L3, new UnsignedDoublewordElement(140)),
 						m(ElectricityMeter.ChannelId.VOLTAGE_L3, new UnsignedDoublewordElement(142))));
 
 		if (this.config.invert()) {
-			modbusProtocol.addTask(new FC3ReadRegistersTask(512, Priority.LOW, //
-					m(ElectricityMeter.ChannelId.ACTIVE_CONSUMPTION_ENERGY, new UnsignedQuadruplewordElement(512),
-							SCALE_FACTOR_MINUS_1),
-					m(ElectricityMeter.ChannelId.ACTIVE_PRODUCTION_ENERGY, new UnsignedQuadruplewordElement(516),
-							SCALE_FACTOR_MINUS_1)));
+			modbusProtocol.addTasks(//
+					new FC3ReadRegistersTask(512, Priority.LOW, //
```

Comment:
There is quite some duplicated code. We could consider implementing it in such a way - but it's up to you.

```
		var inv = this.config.invert();
		new FC3ReadRegistersTask(512, Priority.LOW, //
				m(inv//
						? ElectricityMeter.ChannelId.ACTIVE_CONSUMPTION_ENERGY //
						: ElectricityMeter.ChannelId.ACTIVE_PRODUCTION_ENERGY), //
						new UnsignedQuadruplewordElement(512), SCALE_FACTOR_MINUS_1),
```

I am merging it now anyway, because the PR is very good the way it is. 👍